src/System Application/App/AI/src/Copilot/CopilotAICapabilities.Page.al

```diff
@@ -59,65 +45,156 @@ page 7775 "Copilot AI Capabilities"
                         Hyperlink('https://go.microsoft.com/fwlink/?linkid=2249575');
                     end;
                 }
-            }
-
-            group(NotAlwaysConnected)
-            {
-                ShowCaption = false;
-                InstructionalText = 'By allowing data movement, you agree to data being stored and processed by the Azure OpenAI Service outside of your environment''s geographic region or compliance boundary.';
-                Visible = ((not WithinGeo) or WithinEuropeGeo) and AllowDataMovement;
-
-                field(DataMovement; AllowDataMovement)
+                field(DataSecurityAndPrivacy; FAQForDataSecurityAndPrivacyLbl)
                 {
-                    ApplicationArea = All;
-                    Caption = 'Allow data movement';
-                    ToolTip = 'Specifies whether data movement across regions is allowed. This is required to enable Copilot in your environment.';
-                    Editable = AllowDataMovementEditable;
+                    ShowCaption = false;
 
-                    trigger OnValidate()
+                    trigger OnDrillDown()
                     begin
-                        UpdateAllowDataMovement();
+                        Hyperlink('https://go.microsoft.com/fwlink/?linkid=2298505');
                     end;
                 }
-
-                field(AOAIServiceLocated; AOAIServiceLocatedLbl)
+                field(DataProcessByAOAI; DataProcessByAOAILbl)
                 {
                     ShowCaption = false;
 
                     trigger OnDrillDown()
                     begin
-                        Hyperlink('https://go.microsoft.com/fwlink/?linkid=2250267');
+                        Hyperlink('https://go.microsoft.com/fwlink/?linkid=2298232');
```

Comment:
Right now the link is broken, which is OK because this is the main branch and will be released in 4 months, but make sure we have a task for PMs to write the docs article. If you plan to backport this, it's even more important that the documentation is there soon.

Minor: I prefer to have the hyperlinks labels as locked variables at the bottom of the page, but that's my personal preference :)

Reply:
Thanks :) We do plan to backport. I have checked with Mike, the document should be published before the backport.